.storybook/main.ts

```diff
@@ -0,0 +1,22 @@
+import type { StorybookConfig } from '@storybook/react-vite';
+
+const config: StorybookConfig = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+
+  addons: [
+    '@storybook/addon-onboarding',
+    '@storybook/addon-essentials',
+    '@chromatic-com/storybook',
+    '@storybook/addon-interactions',
+  ],
+
+  framework: {
+    name: '@storybook/react-vite',
+    options: {},
+  },
+
+  typescript: {
+    reactDocgen: 'react-docgen-typescript',
+  },
+};
+export default config;
```

Comment:
will always be my rec to introduce new tools in separate and isolated PRs

Reply:
I will endeavour to do so in the future.